<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View2DSettingsPanel</class>
 <widget class="QWidget" name="View2DSettingsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="invertSliceOrder">
     <property name="text">
      <string>Invert Slice Order</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="invertWheel">
     <property name="text">
      <string>Invert Mouse Wheel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showAxis">
     <property name="text">
      <string>Show Axis</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
